app/assets/src/components/views/pipeline_viz.scss

```diff
@@ -0,0 +1,51 @@
+@import "~styles/themes/colors";
+@import "~styles/themes/typography";
+
+.pipelineViz {
+  width: 100vw;
```

Comment:
I think you should take out the vw and vh units so that the component isn't bound to always taking up the full screen